pkg/kafka/consumergroup/validators.go

```diff
@@ -0,0 +1,33 @@
+package consumergroup
+
+import (
+	"errors"
+	"regexp"
+
+	"github.com/redhat-developer/app-services-cli/pkg/common/commonerr"
+	"github.com/redhat-developer/app-services-cli/pkg/localize"
+)
+
+const (
+	legalSearchChars = "^[a-zA-Z0-9-]+$"
```

Comment:
What validation for search you have in backend @sknot-rh @MikeEdgar ?

Reply:
Which search filter does this apply to? The `topic` query param?

Reply:
It is for `group-id-filter` param.

Reply:
@rkpattnaik780 , @wtrocki - there is currently no validation for that param.

Reply:
@wtrocki @MikeEdgar @craicoverflow should we restrict special characters for this argument? Passing `*` gets into filesystem.
![Screenshot from 2021-07-09 12-53-06](https://user-images.githubusercontent.com/23582438/125040325-25a1df80-e0b5-11eb-808f-d3af021a3c13.png)

Reply:
@craicoverflow turns out validators wont prevent us from this glitch, user needs to use double quotes. Shall I get rid of validation here altogether?
![Screenshot from 2021-07-09 16-31-47](https://user-images.githubusercontent.com/23582438/125068696-67da1980-e0d3-11eb-988b-1a3020437a12.png)

Reply:
Ah I see. yes do, thanks.